qos-monitor/src/main/resources/log4j2.xml

```diff
@@ -5,7 +5,7 @@
             %d{yyyy-MM-dd HH:mm:ss.SSS} %5p ${hostName} --- [%15.15t] %-40.40c{1.} : %m%n%ex
```

Comment:
about qosmonitor_old.p12: don't add backup files into the repository.

Reply:
done